docs/reference/remap/del.cue

```diff
@@ -3,28 +3,28 @@ package metadata
 remap: functions: del: {
 	arguments: [
 		{
-			name:        "paths"
-			description: "The paths of the fields to delete."
+			name:        "path"
+			description: "The path of the field to delete."
 			required:    true
-			multiple:    true
 			type: ["string"]
 		},
 	]
-	return: ["null"]
+	return: ["any"]
 	category: "Event"
 	description: #"""
-		Removed the fields specified by the given paths from the root `event` object. Multiple fields can be specified.
+		Removed the field specified by the given path from the event object. If the field exists,
+		the field's value is returned by the delete operation; if the field does not exist, `null`
+		is returned.
 		"""#
 	examples: [
```

Comment:
It would be nice to have another example showing the renaming of a field here..

`.field2 = del(.field1)`